packages/mobile/src/utils/image.ts

```diff
@@ -32,3 +40,10 @@ export const saveProfilePicture = async (dataUrl: string): Promise<string> => {
     `file://${RNFS.DocumentDirectoryPath}/profile-${Date.now()}`
   )
 }
+
+export const saveRecipientPicture = async (dataUrl: string, address: string): Promise<string> => {
+  return saveImageDataUrlToFile(
+    dataUrl,
+    `file://${RNFS.DocumentDirectoryPath}/CIP8/pictures/${address}`
```

Comment:
Everything stored within the `DocumentDirectoryPath` will be included in the iOS iCloud backup.

Apple has official guidelines about this and since this data can be downloaded again, it should be stored instead in the Caches directory (`CachesDirectoryPath` in RNFS).

See https://developer.apple.com/icloud/documentation/data-storage/